Module/client/src/components/longDescription.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionContainer, ContactHost, ButtonText, ArrowIcon, ShowButton, Paragraph, BoldParagraph,
+} from '../styles/style';
+
+class LongDescription extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      hidden: true,
+    };
+    // divides the long text into an array of paragraphs
+    this.longDescription = props.longDescription.split('\n\n');
+    this.toggleView = this.toggleView.bind(this);
+  }
+
+  toggleView() {
+    this.setState(prevState => ({ hidden: !prevState.hidden }));
+  }
+
+  render() {
+    const { hidden } = this.state;
+    // gives each paragraph there own styled div
+    const paragraphMap = text => text.map((paragraph) => {
```

Comment:
I would put paragraphMap function and Long together in a function that returns Long, and put Short in a function that returns Short.  Call those in the render function so that we can have a separation of concerns and it'll make the render function a bit shorter. So you would have the one line looking like { hidden ? this.getLong() : this.getShort() }. But this is totally okay too! It is up to you ^^